blog/poc3.md

```diff
@@ -0,0 +1,127 @@
+
+
+The message payload is dispatched to message handlers by the recipient nodes and dispatched to subscribers via a JSON-RPC API (typically over web-socket) depending on the topic.
+Nodes can communicate privately with each other  even if they are not each others' peers.
+Forward secrecy is provided by a 'Diffie-Hellmann handshake' and ephemeral keys.
+As a result PSS can maintain arbitrary dark subnetworks in swarm. These subnetworks indentified by the topics can even handle devp2p protocols.
+Not only can pss be used for slow-scale chat messaging, but also for secure file transfer, notifications and collaborative protocols like [CRASH proof of custody]() used in storage incentivisation.
+
+PSS is fully featured yet experimental on the new POC3 network. We are collaborating closely with companies and projects that want to use pss to build mid-layer infrastructure. [Mainframe](http://mainframe.com) is building a slack-alternative collaborative group
+ communications tool ([Onyx](https://blog.mainframe.com/mainframe-pre-alpha-release-fca532317111)) and their web3 SDK, and [Status](http://status.im) have expressed interest in building it into their mobile chat.
+
+## Mutability
+
+Using hashes as content addresses allows us to view hashes as addresses at which content is expected to be found.
+Since hashes can be assumed to be collision free, they are bound to one specific version of a content. Hash addressing is therefore immutable in the strong sense that you cannot even express mutable content: "changing the content changes the hash".
+
+Users of the web, however, are accustomed to mutable resources, looking up domains and expect to see the most up to date version of the 'site'. Mutable resources are made possible by the ethereum name service (ENS) and Mutable Resource Updates (MRU).
+The ENS is a smart contract on the ethereum blockchain which enables domain owners to register a content reference to their domain.
+Using ENS for domain name resolution, the url scheme provides
+content retrieval based on mnemonic (or branded) names, much like the DNS of the world wide web, but without servers.
+Mutable Resource Update is, on the other hand, an off-chain solution for communicating updates to a resource, it offers cheaper and potentially faster updates than ENS, yet the updates can be consolidated on ENS by any third party willing to pay for the transaction.
+
+Here is how it works: a publisher creates a mutable resource at time `t` for the name "my daily tweets", that she intends to update every day once (note this in itself not a promise). By hashing together `ID := SHA3(publisher_pubkey, name, start, period)`, we obtain a so called metadata chunk.
```

Comment:
not sure if i'd go into such technical detail within the scope of this blog post